packages/sdk/src/peer_manager/peer_manager.ts

```diff
@@ -117,13 +118,20 @@ export class PeerManager {
 
     for (const peer of connectedPeers) {
       const hasProtocol = this.hasPeerProtocol(peer, params.protocol);
-      const hasSamePubsub = await this.connectionManager.isPeerOnTopic(
-        peer.id,
-        params.pubsubTopic
-      );
+
+      if (params.pubsubTopic) {
```

Comment:
nit: why this check? from `GetPeersParams` type - `pubsubTopic` must be present

Reply:
I assume pre-existing, will see if I can clean up.